docs/debuginfod.md

```diff
@@ -0,0 +1,45 @@
+# debuginfod client
+
+In order to symbolize ingested profiles, Parca needs to have debug information
+for the binaries that are being profiled. Debug information
+typically comprises ELF object files, DWARF debug data, and source code
```

Comment:
It's best to use simple language in docs as most readers are not native, I think people might stumble over "typically comprises" here. I think maybe we can say "can be"? I think it would also be good to say something like "Debug information, also referred to as _debuginfos_". And skip the ending of the sentence "required for symbolization", I think we already say this in the second part of the first sentence.